invenio/templates/configurations/haproxy.yaml

```diff
@@ -46,6 +46,8 @@ data:
       http-request deny if is_blocked_ip OR is_blocked_ua
       errorfiles custom_errors
 
+      {{ .Values.haproxy.extra_frontend_public_http_request| indent 6 }}
```

Comment:
if you put `indent 6`, I think you have to put this all to the left

Reply:
Tested this and the changes are also OK. We're merging for now since it's working.